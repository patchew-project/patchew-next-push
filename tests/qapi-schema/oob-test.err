tests/qapi-schema/oob-test.qapi:2: 'allow-oob' of command 'oob-command-1' should only use true value
