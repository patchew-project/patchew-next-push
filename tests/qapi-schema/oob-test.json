# Some Out-Of-Band specific tests
{ 'command': 'an-oob-command', 'allow-oob': true }
