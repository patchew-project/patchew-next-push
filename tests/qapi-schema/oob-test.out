object q_empty
enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
module oob-test.json
command an-oob-command None -> None
   gen=True success_response=True boxed=False oob=True
